python/paddle/profiler/profiler_statistic.py

```diff
@@ -698,9 +698,12 @@ def _build_table(statistic_data,
                  sorted_by=SortedKeys.CPUTotal,
                  op_detail=True,
                  thread_sep=False,
+                 views=None,
```

Comment:
same problem as above. 
But if this function is not public API and just used by internal calls, that's OK